tests/python/contrib/test_arm_compute_lib/test_dense.py

```diff
@@ -154,7 +154,7 @@ def _get_expected_codegen(shape, weight_shape, units, dtype, has_bias=False):
             {
                 "op": "const",
                 "name": "",
-                "attrs": {"shape": [[[weight_shape[0]]]], "dtype": [[bias_dtype]]},
+                "attrs": {"shape": [[[1, weight_shape[0]]]], "dtype": [[bias_dtype]]},
```

Comment:
cc @lhutton1 Any idea why one of arm compute lib tests failed with this PR? https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/PR-8669/7/pipeline/

Reply:
PS I'm unsure whether these changes actually break our implementation later down the line in the runtime. It looks like the runtime tests for ACL aren't running on the Arm CI for some reason in order to test this. @u99127 @leandron do we need to re-add ACL to the Jenkinsfile after #8400?

Reply:
I think it was re-enabled in https://github.com/apache/tvm/pull/8573?

Reply:
It looks like not all the changes were restored in that PR, #8734 should fix this